backend/routes/db.py

```diff
@@ -103,32 +99,20 @@ def get_cset_members_items(
         ...
         item: True if its an expression item, else false
         csm: false if not in concept set members
+    todo: should check that column names are valid cols in concept_set_members but probably never use this option anyway
+    todo: column/columns interpolation probably unsafe in regards to SQL injection
     """
-    if column:
-        # should check that column names are valid columns in concept_set_members
-        # but probably never use this option anyway
-        columns = [column]
-    if not columns:
-        columns = ['*']
+    if column and columns:
+        raise ValueError('Cannot specify both columns and column')
+    columns = [column] if column else columns if columns else ['*']
 
     with get_db_connection() as con:
-        # SELECT DISTINCT {', '.join(columns)}
-        (pstr, params) = sql_in_safe(codeset_ids)
+        pstr, params = sql_in_safe(codeset_ids)
         query = f"""
-            SELECT DISTINCT *
+            SELECT DISTINCT {', '.join(columns)}
```

Comment:
## Updated `get_cset_members_items()`
- Added filtration of columns
 - Made sure new test case passes
 - Made sure old test case still passes
- Minor: Added todo for refactoring better against SQL injection
- Minor: Raised error if passing conflicting params
- Minor: Refactoring; removed dead / redundant code